provers/risc0/builder/src/main.rs

```diff
@@ -6,7 +6,12 @@ use std::path::PathBuf;
 fn main() {
     let pipeline = Risc0Pipeline::new("provers/risc0/guest", "release");
     pipeline.bins(
-        &["risc0-guest", "risc0-aggregation"],
+        &[
+            "risc0-guest",
+            "risc0-aggregation",
+            "risc0-guest-mock",
```

Comment:
maybe better to use feature to separate build.
please also check if the image id output in compile log is clear or not as we need give that id to client to config the contract.